docs/authoring-components.md

```diff
@@ -250,39 +246,35 @@ be used to build out lint tools to enforce proper adapter shape. This example sh
 as making use of our `MDCFoundation` class, which is the base class which all foundations inherit
 from.
 
-```js
+```ts
 class RedblueToggleFoundation extends MDCFoundation {
   static get defaultAdapter() {
     return {
       getAttr: (/* attr: string */) => /* string */ '',
-      setAttr: (/* attr: string, value: string */) => {},
-      addClass: (/* className: string */) => {},
-      removeClass: (/* className: string */) => {},
-      setToggleColorTextContent: (/* textContent: string */) => {},
-      registerInteractionHandler: (/* type: string, handler: EventListener */) => {},
-      deregisterInteractionHandler: (/* type: string, handler: EventListener */) => {}
+      setAttr: (/* attr: string, value: string */) => undefined,
```

Comment:
Replace Closure Doc comments with TypeScript

Reply:
I replaced it with:
```
      getAttr: (attr: string) => '',
      setAttr: (attr: string, value: string) => undefined,
      addClass: (className: string) => undefined,
      removeClass: (className: string) => undefined,
      setToggleColorTextContent: (textContent: string) => undefined,
      registerInteractionHandler: (type: string, handler: EventListener) => undefined,
      deregisterInteractionHandler: (type: string, handler: EventListener) => undefined
```

Although in our actual implementation we don't have unused arguments. Should I just remove them to look like:

```
      getAttr: () => '',
      setAttr: () => undefined,
      addClass: () => undefined,
      removeClass: () => undefined,
      setToggleColorTextContent: () => undefined,
      registerInteractionHandler: () => undefined,
      deregisterInteractionHandler: () => undefined,
```

Reply:
I would probably leave them to make it clear that they exist, even if we're not using them.